src/Momento.Sdk/Exceptions/SdkException.cs

```diff
@@ -1,13 +1,65 @@
 ﻿using System;
+using Grpc.Core;
 
 namespace Momento.Sdk.Exceptions;
 
+public enum MomentoErrorCode {
+  INVALID_ARGUMENT_ERROR,
+  UNKNOWN_SERVICE_ERROR,
+  ALREADY_EXISTS_ERROR,
+  NOT_FOUND_ERROR,
+  INTERNAL_SERVER_ERROR,
+  PERMISSION_ERROR,
+  AUTHENTICATION_ERROR,
+  CANCELLED_ERROR,
+  LIMIT_EXCEEDED_ERROR,
+  BAD_REQUEST_ERROR,
+  TIMEOUT_ERROR,
+  SERVER_UNAVAILABLE,
+  CLIENT_RESOURCE_EXHAUSTED,
+  FAILED_PRECONDITION_ERROR,
+  UNKNOWN_ERROR
+}
```

Comment:
i know you may have this in a branch already and it's fine for it to be in a PR, but we should see if we can add some comment strings here that make these a little more clear.

Reply:
This is indeed on a branch that will be rolling out as a PR as soon as I merge this one.